src/wcm.cpp

```diff
@@ -1697,3 +1698,33 @@ std::shared_ptr<wf::config::section_t> WCM::get_config_section(Plugin *plugin)
 #endif
     return nullptr;
 }
+
+std::string WCM::find_icon(const std::string & name)
+{
+    // first try to find it in the user's local folders
+    std::string icon_path;
+    std::string xdg_data_dir;
+    char *c_xdg_data_dir = std::getenv("XDG_DATA_HOME");
+    char *c_user_home    = std::getenv("HOME");
+
+    if (c_xdg_data_dir != NULL)
+    {
+        xdg_data_dir = c_xdg_data_dir;
+    } else
+    {
+        if (c_user_home != NULL)
+        {
+            xdg_data_dir = (std::string)c_user_home + "/.local/share/";
+        }
+    }
+
+    if (xdg_data_dir.size())
```

Comment:
Please use `std::string::empty` method instead of comparing `size` with zero.

Reply:
done